src/Analyzers/AnalyzerRunner.cs

```diff
@@ -11,20 +11,29 @@ namespace Microsoft.CodeAnalysis.Tools.Analyzers
 {
     internal partial class AnalyzerRunner : IAnalyzerRunner
     {
+        private readonly bool _includeComplilerDiagnostics;
+
+        public AnalyzerRunner(bool includeCompilerDiagnostics)
+        {
+            _includeComplilerDiagnostics = includeCompilerDiagnostics;
```

Comment:
This should open up the possibility of running fixers that have no analyzer because they work against compiler diagnostics.